src/front-end/src/components/MenteeDisplay/MenteeDisplay.test.js

```diff
@@ -66,7 +66,8 @@ test('render no email fields when input suggested mentees', () => {
   const suggested = {
     menteeUid: '',
     name: 'Eric',
-    // this field is set empty because in suggestedMentee.js it is set empty as well.
+    /* this field is set empty because in suggestedMentee.js it is set empty 
+    as well. */
```

Comment:
The console displayed a warning since the comment was >80 characters so I made it a block comment